### PR TITLE
Search

### PR DESCRIPTION
**Pull request by PeterSafwatHBK**

### CODE REVIEW

File: src/sockets/middlewares.ts
Comment (amir-kedis) [inline src/sockets/middlewares.ts:1]: why change socket handshake, search doesn't use sockets?

**Review by amir-kedis — COMMENTED** [review]
Check the socket auth as it may break other parts. @AhmedHamed3699 @PeterSafwatHBK

**Comment by PeterSafwatHBK** [comment]
I wanted to send some messages to search for , but the handshake line gave
an error and they told me to change it and t worked.

**Comment by PeterSafwatHBK** [comment]
If you want to change it back ok, but I doubt messaging will work.

**Pull request merged**